b2share/modules/communities/api.py

```diff
@@ -26,20 +26,24 @@
 
 from __future__ import absolute_import
 
-import sqlalchemy
 import uuid
 
 from invenio_db import db
 from jsonpatch import apply_patch
+
+import sqlalchemy
```

Comment:
Why import the whole package when you are also importing the methods individually?

Reply:
Try if this can be removed in the future